internal/handlers/common/params.go

```diff
@@ -258,3 +258,44 @@ func addNumbers(v1, v2 any) (any, error) {
 		return nil, errUnexpectedLeftOpType
 	}
 }
+
+// GetPositiveNumber returns doc's value for key or protocol error for missing or invalid parameter.
+func GetPositiveNumber(document *types.Document, key string, isOptional bool) (int32, error) {
+	v, err := document.Get(key)
+	if err != nil {
+		if isOptional {
```

Comment:
Parameter `isOptional` is always true. I think it should be removed and the logic should be changed accordingly. Also it could be renamed from `GetPositiveNumber` to `GetOptionalPositiveNumber`

Reply:
fixed, I decided to return nil as we return in `GetOptionalParam` method